Automattic/wp-calypso
# wp-admin Site Default: Refactor design to use radio buttons

Fixes https://github.com/Automattic/dotcom-forge/issues/4259

## Proposed Changes

Updates the styling for the 'Admin interface style' card:

<img width="790" alt="CleanShot 2023-10-20 at 06 30 24@2x" src="https://github.com/Automattic/wp-calypso/assets/36432/867dd5ce-e0a9-41cf-84dd-6b5ee388a6fc">

Also adds notices and a Tracks event.

## Testing Instructions

1. Navigate to Hosting Configuration.
2. Verify that the radio buttons work as expected, and that the Tracks events fire.
